extensions/udapdr/README.md

```diff
@@ -0,0 +1,64 @@
+# UDAPDR
+
+## Overview
+
+This repository includes the code and datasets for the experiments in [UDAPDR: Unsupervised Domain Adaptation via LLM Prompting and Distillation of Rerankers](https://arxiv.org/abs/2303.00807).
+
+## Installation
+
+Run the following commands to setup a conda environment:
+
+````
+conda create --name udapdr --file requirements.txt
+conda activate udapdr
```

Comment:
Are the packages in requirements.txt  in addition to those for primeqa ?

Reply:
They need to install PrimeQA as well. I'll mention that in the ReadMe